Sources/Workspace/Workspace.swift

```diff
@@ -435,13 +425,13 @@ public class Workspace {
         // If the dependency isn't in edit mode, we can't unedit it.
         case .checkout: 
             throw WorkspaceOperationError.dependencyNotInEditMode
-        case .edited:
+        case .edited(nil):
```

Comment:
It is, the `path != nil` below is un-necessary. We don't want a managed edited dependency to overwrite the parameter's `forceRemove`.